go.mod

```diff
@@ -42,12 +42,14 @@ require (
 	github.com/spf13/cobra v1.2.1
 	github.com/spf13/pflag v1.0.5
 	github.com/tidwall/gjson v1.11.0
-	golang.org/x/crypto v0.0.0-20211117183948-ae814b36b871
+	github.com/vishvananda/netlink v1.1.1-0.20210330154013-f5de75959ad5
+	github.com/vishvananda/netns v0.0.0-20210104183010-2eb08e3e575f
+	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/sys v0.0.0-20211117180635-dee7805ff2e1
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	gotest.tools/v3 v3.0.3
 )
 
-// Temporary fork for avoiding importing patent-protected code: https://github.com/hashicorp/golang-lru/issues/73
+// Temporary fork for avoiding importing patent-protected code: https://github.com/hashicorp/g20211117183948-ae814b36b871olang-lru/issues/73
```

Comment:
????

Reply:
sorry!  I don't know where it comes from ^^